src/Raven.Server/Documents/PeriodicBackup/Aws/RavenAwsS3Client.cs

```diff
@@ -43,9 +47,19 @@ public RavenAwsS3Client(S3Settings s3Settings, Progress progress = null, Logger
             if (string.IsNullOrWhiteSpace(s3Settings.AwsAccessKey))
                 throw new ArgumentException("AWS Access Key cannot be null or empty");
 
-            if (string.IsNullOrWhiteSpace(s3Settings.AwsRegionName))
-                throw new ArgumentException("AWS Region Name cannot be null or empty");
-
+            if (string.IsNullOrWhiteSpace(s3Settings.CustomS3ServerUrl))
+            {
+                if (string.IsNullOrWhiteSpace(s3Settings.AwsRegionName))
+                    throw new ArgumentException("AWS region Name cannot be null or empty");
+                AwsRegion = s3Settings.AwsRegionName.ToLower();
+            }
+            else
+            {
+                if (string.IsNullOrWhiteSpace(s3Settings.AwsRegionName) == false)
+                    throw new ArgumentException($"There is no need for region in custom S3 server: AwsRegionName({s3Settings.AwsRegionName})");
+                _customS3ServerUrl = new Uri(s3Settings.CustomS3ServerUrl);
+            }
```

Comment:
This seems to indicate that the region name isn't required for custom S3, but is that the case?
I don't think that this is true, see: https://developers.digitalocean.com/documentation/spaces/